Jenkinsfile

```diff
@@ -178,6 +179,32 @@ pipeline {
             githubNotify(context: "${env.GITHUB_CHECK_ITS_NAME}", description: "${env.GITHUB_CHECK_ITS_NAME} ...", status: 'PENDING', targetUrl: "${env.JENKINS_URL}search/?q=${env.ITS_PIPELINE.replaceAll('/','+')}")
           }
         }
+        stage('Release') {
+          options { skipDefaultCheckout() }
+          when {
+            beforeAgent true
+            tag pattern: '@elastic/apm-rum@\\d+\\.\\d+\\.\\d+$', comparator: 'REGEXP'
+          }
+          stages {
+            stage('Opbeans') {
+              environment {
+                REPO_NAME = "${OPBEANS_REPO}"
+              }
+              steps {
+                deleteDir()
+                dir("${OPBEANS_REPO}"){
+                  git credentialsId: 'f6c7695a-671e-4f4f-a331-acdce44ff9ba',
```

Comment:
don't we need to encrypt this token? or it it read only?

Reply:
Already encrypted, that's the internal hash id, nothing to be worried about